EIPS/eip-1559.md

```diff
@@ -48,15 +48,15 @@ The transition to this gas price system will occur in two phases, in the first p
 * `FINAL_FORK_BLKNUM`: `INITIAL_FORK_BLKNUM + MIGRATION_DURATION_IN_BLOCKS`
 * `BLOCK_GAS_TARGET`: The gas limit field set in the block header, controlled by miner voting.
 * `EIP1559_INITIAL_GAS_TARGET`: `BLOCK_GAS_TARGET / 2`
-* `LEGACY_INITIAL_GAS_LIMIT`: `BLOCK_GAS_TARGET - EIP1559_GAS_TARGET`
+* `LEGACY_INITIAL_GAS_LIMIT`: `BLOCK_GAS_TARGET / 2`
```

Comment:
The "bug" here was that `BLOCK_GAS_TARGET - EIP1559_GAS_TARGET` should have been `BLOCK_GAS_TARGET - EIP1559_INITIAL_GAS_TARGET`.  In theory, by having it that way it would be "easier" to update the EIP with a different initial distribution (e.g., if we wanted to start with 10% of block switched over initially).